third_party/payjs/README.amp

```diff
@@ -0,0 +1,9 @@
+URL: https://github.com/google/payjs
+License: Apache 2.0
+License File: https://github.com/google/payjs/blob/master/LICENSE
+
+Description:
+Copy of Google Pay integration library
+
+Local Modifications:
+Removed bazel build files
```

Comment:
This doesn't seem to be a big deal. Can we just import it as a dependency in package.json? We have an auto-upgrade bot for dependencies.

Reply:
third party changes cl https://github.com/ampproject/amphtml/pull/24871